openbas-api/src/main/resources/application.properties

```diff
@@ -157,6 +157,9 @@ executor.tanium.action-group-id=4
 executor.tanium.windows-package-id=
 executor.tanium.unix-package-id=
 
+# Executor OpenBAS
+executor.openbas.version=local
+
```

Comment:
I feel this config's current name can be confusing; I'd alter its use further though.

As we discussed, the aim of this config is to allow bypassing the locally contained agent binaries and have OpenBAS serve a binary directly from Artifactory given a specific version.

I'd suggest changing the naming semantics:
```suggestion
# Executor OpenBAS

# valid values: local | repository
# default: local
executor.openbas.agent.binaries.origin=

# if executor.openbas.agent.binaries.origin is set to "local",
# this config is ignored
# default: the OpenBAS instance's version
executor.openbas.agent.binaries.version=

```
Leaving them both blank, hence the default behaviour should just be locally hosted binaries.

I know it's two configs rather than a single one, but I'm uncomfortable with a single config that triggers multiple behaviours (here that would be changing the origin, and the version of the binaries).